src/main/java/org/folio/inventory/dataimport/handlers/actions/ReplaceInstanceEventHandler.java

```diff
@@ -215,6 +215,7 @@ private void processInstanceUpdate(DataImportEventPayload dataImportEventPayload
             if (instanceToUpdate.getSource().equals(MARC.getValue())) {
               setExternalIds(targetRecord, instance);
               AdditionalFieldsUtil.remove035FieldWhenRecordContainsHrId(targetRecord);
+              AdditionalFieldsUtil.normalize035(targetRecord);
```

Comment:
please, place this method call before instance mapping so that the normalized value will be set to instance